tools/clang/lib/SPIRV/SPIRVEmitter.cpp

```diff
@@ -6525,6 +6564,7 @@ bool SPIRVEmitter::emitEntryFunctionWrapper(const FunctionDecl *decl,
   // Initialize all global variables at the beginning of the wrapper
   for (const VarDecl *varDecl : toInitGloalVars) {
     const auto id = declIdMapper.getDeclResultId(varDecl);
+    assert(id);
```

Comment:
the above suggestion will eliminate the need for this assert.

Reply:
Done